[DOC-70] Do not merge till [2.20.3] Commvault integration with YBA (#21002)
yugabyte/yugabyte-db: docs/content/preview/integrations/commvault.md

---
title: Commvault
linkTitle: Commvault
description: Use Commvault to back up and restore your YugabyteDB database.
menu:
  preview_integrations:
    identifier: commvault
    parent: integrations-platforms
    weight: 571
type: docs
---

Commvault is a data protection and data management platform that provides an agent-based solution to support the following operations in [YugabyteDB Anywhere (YBA)](../../yugabyte-platform/):

- Data protection for YugabyteDB universes in AWS and Azure for xCluster deployment.
- Backup and restore of YSQL and YCQL databases.

> - Job monitoring and manageability from the Commvault's [Command center](https://www.commvault.com/platform/command-center).

Reviewer: ```suggestion
- Job monitoring and manageability from the Commvault [Command Center](https://www.commvault.com/platform/command-center).
```

## Connect

> To use Commvault with YugabyteDB, you need to configure YBA on Commvault's Command center.

Reviewer: ```suggestion
To use Commvault with YugabyteDB, you need to configure YBA on the Commvault Command Center.
```

> For more information, refer to Commvault's documentation on [Getting Started with YugabyteDB](https://documentation.commvault.com/2024/essential/getting_started_with_yugabytedb.html).

Reviewer: ```suggestion
For more information, refer to [Getting Started with YugabyteDB](https://documentation.commvault.com/2024/essential/getting_started_with_yugabytedb.html) in the Commvault documentation.
```
